web/src/engine/predictive-text/worker-thread/src/main/correction/context-tokenization.ts

```diff
@@ -47,6 +48,16 @@ export class ContextTokenization {
     return this.tokens[this.tokens.length - 1];
   }
 
+  /**
+   * Returns plain-text strings representing the most probable representation for all
+   * tokens represented by this tokenization instance.
+   */
+  get sourceText() {
+    return this.tokens
+      .filter(token => token.sourceText !== null)
```

Comment:
Using `sourceText` here contradicts the [comment](https://github.com/keymanapp/keyman/pull/14756/files#diff-00a35722bff4fbfe56f2f0923358b2961e6e096fb21e38b9c5933b2e5906addcR141) in `ContextToken.sourceText`.

Reply:
Right, I'll need to change that - it certainly doesn't stay that way.